appcues/src/main/java/com/appcues/analytics/AnalyticsQueueProcessor.kt

```diff
@@ -16,22 +16,29 @@ internal class AnalyticsQueueProcessor(
     private val appcuesCoroutineScope: AppcuesCoroutineScope,
     private val repository: AppcuesRepository,
     private val experienceRenderer: ExperienceRenderer,
+    // this is the background analytics processing queue - 10 sec batch
     private val analyticsQueueScheduler: QueueScheduler,
+    // this is the immediate processing queue - 50ms batch to group identify with immediate subsequent updates
+    private val priorityQueueScheduler: QueueScheduler,
```

Comment:
@andretortolano it was simpler for review to not rename old things, but I think I might take another pass over just to rename, not change any logic. As I think having the old stuff named `analyticsQueueScheduler` and `pendingActivity` is not really ideal now, maybe something like `backgroundQueueScheduler` and `backgroundQueue` -- I just want to make it more clear why there are two paths of things here and which is background vs priority.